.github/workflows/main.yml

```diff
@@ -229,38 +229,130 @@ jobs:
+          ### Variables ###
+          export SSO_PROTOCOL=https
+          export GATEWAY_PROTOCOL=https
+          export GLOBAL_GATEWAY_DOMAIN=$CLUSTER_NAME.$CLUSTER_DOMAIN
+          export GATEWAY_HOST=gateway-$PREVIEW_NAME.$GLOBAL_GATEWAY_DOMAIN
+          export SSO_HOST=identity-$PREVIEW_NAME.$GLOBAL_GATEWAY_DOMAIN
+
+          ### Functions ####
+
+          # It evaluates the function passed as a parameter
+          # if the result is different than 0 it waits
+          # for 5 seconds and re-evaluates it again.
+          #
+          wait_until_true () {
+            local attempt_counter=0
+            local max_attempts=50
+
+            until "$@"
+            do
+              if [ ${attempt_counter} -eq ${max_attempts} ]
+              then
+                echo "Max attempts reached"
+                break
+              fi
+
+              printf '.'
+              attempt_counter=$((attempt_counter+1))
+              sleep 5
+            done
+          }
+
+          # It makes an HTTP request to the health check of the
+          # modeling, query and runtime-bundles API returning 0
+          # in case their pods are ready and a value different
+          # than 0 otherwise.
+          #
+          check_services_up () {
             curl --silent --head --fail $GATEWAY_PROTOCOL://$GATEWAY_HOST/modeling-service/actuator/health > /dev/null 2>&1 && \
             curl --silent --head --fail $GATEWAY_PROTOCOL://$GATEWAY_HOST/rb/actuator/health > /dev/null 2>&1 && \
             curl --silent --head --fail $GATEWAY_PROTOCOL://$GATEWAY_HOST/query/actuator/health > /dev/null 2>&1
-          do
-            if [ ${attempt_counter} -eq ${max_attempts} ]
-            then
-              echo "Max attempts reached"
-              break
-            fi
+          }
 
-            printf '.'
-            attempt_counter=$((attempt_counter+1))
-            sleep 5
-          done
+          ### Main ###
+
+          echo "Waiting for services to be up..."
+          wait_until_true check_services_up
 
+          # Showing the deployment result
           kubectl get all -n $PREVIEW_NAME
 
+          # Launching the tests
           make test/modeling-acceptance-tests
+
+      - name: Lanunch runtime acceptance test (not kafka partitioned)
+        if: ${{ matrix.messaging-broker != 'kafka' && matrix.messaging-partitioned != 'partitioned' }}
+        run: |
+          ### Variables ###
+          export SSO_PROTOCOL=https
+          export GATEWAY_PROTOCOL=https
+          export GLOBAL_GATEWAY_DOMAIN=$CLUSTER_NAME.$CLUSTER_DOMAIN
+          export GATEWAY_HOST=gateway-$PREVIEW_NAME.$GLOBAL_GATEWAY_DOMAIN
+          export SSO_HOST=identity-$PREVIEW_NAME.$GLOBAL_GATEWAY_DOMAIN
+
+          make test/runtime-acceptance-tests
+
+      - name: Increase the consumers number and launch runtime acceptance test (only kafka partitioned)
+        if: ${{ matrix.messaging-broker == 'kafka' && matrix.messaging-partitioned == 'partitioned' }}
```

Comment:
With the current configuration, we are running only the modeling tests for `rabbitmq-partitioned` and `kafka-non-partiotioned`. We need to have runtime acceptance tests running for those configurations as well.

Reply:
Talking with @erdemedeiros we decided to modify the subject of the test of the code. So this comment is not applicable anymore.